cura/Settings/MachineManager.py

```diff
@@ -132,6 +117,19 @@ def __init__(self, parent = None):
         if containers:
             containers[0].nameChanged.connect(self._onMaterialNameChanged)
 
+        # NEW
+        self._material_manager = self._application._material_manager
+        self._material_manager.materialsUpdated.connect(self._onMaterialsUpdated)
+
+    def _onMaterialsUpdated(self):
+        # When the materials lookup table gets updated, it can mean that a material has its name changed, which should
+        # be reflected on the GUI. This signal emission makes sure that it happens.
+        self.rootMaterialChanged.emit()
+
+    ### NEW
```

Comment:
Remove `### NEW`. In a few years it won't be new any more.

Reply:
Removed in 632be1132